modules/apps/calendar/calendar-web/src/main/java/com/liferay/calendar/web/internal/info/item/provider/CalendarBookingInfoItemCapabilitiesProvider.java

```diff
@@ -26,6 +28,10 @@ public class CalendarBookingInfoItemCapabilitiesProvider
 
 	@Override
 	public List<InfoItemCapability> getInfoItemCapabilities() {
+		if (!FeatureFlagManagerUtil.isEnabled("LPS-201853")) {
```

Comment:
Error on console
```
2023-12-18 16:06:47.001 ERROR [SystemExecutorServiceUtil-8-com.liferay.portal.spring.extender.internal.configuration.ServiceConfigurationInitializer-com.liferay.template.service][FeatureFlagsBag:90] Feature flag LPS-201853 is not available for company 0
```

It's asking for a companyId, so **maybe** smoke test will fail

Reply:
You're right :facepalm:

Reply:
Changes squashed in https://github.com/liferay-lima/liferay-portal/pull/5018/commits/2407c67088fa1bb458b2865b0514b80f00ea670e